core/src/forwarding_stage.rs

```diff
@@ -496,16 +570,37 @@ mod tests {
         super::*,
         crossbeam_channel::unbounded,
         packet::PacketFlags,
-        solana_net_utils::bind_to_unspecified,
         solana_perf::packet::{Packet, PacketBatch},
         solana_pubkey::Pubkey,
         solana_runtime::genesis_utils::create_genesis_config,
         solana_sdk::{hash::Hash, signature::Keypair, system_transaction},
+        std::sync::{Arc, Mutex},
     };
 
-    impl ForwardAddressGetter for ForwardingAddresses {
-        fn get_forwarding_addresses(&self, _protocol: Protocol) -> ForwardingAddresses {
-            self.clone()
+    #[derive(Clone)]
```

Comment:
should this be dev-context-only-utils or maybe even in mod tests?

Reply:
it is in module `tests` already, do you propose to move to some separate file?

Reply:
sorry i missed that!